### ISSUE TITLE
TagLib: Using wrong XSD

### ISSUE THREAD

**Issue by melloware** [primefaces/primefaces#12799]
### Describe the bug

cc @bvfalcon 

Reported by @bvfalcon the primefaces.taglib.xml is using the wrong XSD.

It should be https://jakarta.ee/xml/ns/jakartaee/web-facelettaglibrary_2_0.xsd

### PrimeFaces edition

Community

### PrimeFaces version

14.0.6

### Theme

ALL

### JSF implementation

Mojarra

### JSF version

2.3

### Java version

11

### Browser(s)

ALL

**Comment by bvfalcon** [comment]
By the way: one of attributes (pattern) contains incorrect type declaration: `<type>String</type>`.  Must be <type>java.lang.String</type>

**Comment by melloware** [comment]
Thanks fixed in PR!

**Comment by aripddev** [comment]
Probably after this commit I started getting the following error.  So the deployment has been failed.

```
Critical error during deployment: 
com.sun.faces.config.ConfigurationException: java.util.concurrent.ExecutionException: com.sun.faces.config.ConfigurationException: Unable to parse document 'jar:file:/.../target/webapp/WEB-INF/lib/primefaces-15.0.0-SNAPSHOT-jakarta.jar!/META-INF/primefaces.taglib.xml': Unknown Schema version: 2.0
```

In the `primefaces.taglib.xml`, `version="2.0"` should be changed to `version="2.3"`

**Comment by melloware** [comment]
werid i don't even remember changing that to 2.0 i will fix it.

**Comment by melloware** [comment]
Thanks @aripddev i just committed the fix on 15.0.0 and 14.X
